modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -36,10 +35,61 @@ import {
   XShareWithNtilde,
   YShare,
   YShareWithNtilde,
+  SerializedNtilde,
+  DeserializedNtilde,
 } from './types';
 
 const _5n = BigInt(5);
 
+/**
+ * Deserializes a challenge and it's proofs from hex strings to bigint
+ */
+export function deserializeNtilde(challenge: SerializedNtilde): DeserializedNtilde {
```

Comment:
Moved due to circular dependencies. Also makes sense to keep this closer to the types file that defines these two types.